slave/vagrant/fabfile.py

```diff
@@ -42,6 +42,11 @@ def install(index, password, master='build.staging.clusterhq.com'):
 yum install -y https://kojipkgs.fedoraproject.org//packages/kernel/${KV}/${SV}/${ARCH}/kernel-devel-${UNAME_R}.rpm  # noqa
 """)
 
+    run("""
+ARCH=$(uname -m)
+yum install -y https://clusterhq.s3.amazonaws.com/repo/fedora/20/${ARCH}/phantomjs-1.9.8-1.x86_64.rpm  # noqa
```

Comment:
I don't think it makes to use this long path. It isn't a yum repository, for example. Also, the files in S3 didn't have wide enough permissions to work. I've addressed both of these things.